### PR TITLE
feat(YouTube - Player overlay): Change player background overlay opacity

### PR DESCRIPTION
**Pull request by johnconner122**
Depends on:

 - https://github.com/ReVanced/revanced-integrations/pull/449

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
Can you resolve the merge conflicts?

**Comment by johnconner122** [comment]
> Can you resolve the merge conflicts?

It says, only people with write access can resolve the merge conflict.

**Comment by oSumAtrIX** [comment]
You have to resolve it in your branch.

**Comment by johnconner122** [comment]
Screenshot_2023-08-03-07-44-29-660-edit_com brave browser

**Comment by oSumAtrIX** [comment]
Yes, you have to resolve it in your own branch, merge upstream

**Comment by johnconner122** [comment]
> Yes, you have to resolve it in your own branch, merge upstream

I used `revanced:main` as upstream repository?

**Comment by oSumAtrIX** [comment]
I am currently debating if the complexity of this patch is worth adding, considering it is not that useful.

File: src/main/kotlin/app/revanced/patches/youtube/layout/player/overlay/annotations/PlayerOverlayPatchCompatibility.kt
[the remark this answers is not in the document]
Comment (johnconner122) [reply src/main/kotlin/app/revanced/patches/youtube/layout/player/overlay/annotations/PlayerOverlayPatchCompatibility.kt:1]: I don't think it needs version constraint, I have checked that even in youtube `v17.x.x`, `resource-id` is same.

**Pull request merged**